library/Freckle/App/Database.hs

```diff
@@ -309,4 +312,5 @@ postgresConnectionString PostgresConnectionConf {..} password =
     , "user=" <> pccUser
     , "password=" <> password
     , "dbname=" <> pccDatabase
+    , "options=-csearch_path=" <> pccSchema
```

Comment:
It feels inconsistent to handle the timeout by executing a command on the connection at startup, but handle the schema by pushing `options` into the connection string. Can we make it so we do them by the same method? I don't mind which way -- I assume we can set a timeout via `options` or we can set the schema via executing a command.

Reply:
I didn't realize that's how timeouts are handled. I think this should be done this same way as that